gosnmp.go

```diff
@@ -22,12 +22,12 @@ import (
 )
 
 const (
-	// maxOids is the maximum number of oids allowed in a Get()
-	maxOids = 60
-
 	// Base OID for MIB-2 defined SNMP variables
 	baseOid = ".1.3.6.1.2.1"
 
+	// Default MaxOids value, 60 is reasonable
```

Comment:
Putting this back, albeit on a different line (sorry), for use in initialising a default